client/my-sites/sharing/connections/account-dialog.jsx

```diff
@@ -20,24 +20,37 @@ class AccountDialog extends Component {
 		isVisible: PropTypes.bool,
 		onAccountSelected: PropTypes.func,
 		service: PropTypes.object,
+		translate: PropTypes.func,
+		warningNotice: PropTypes.func,
 	};
 
 	static defaultProps = {
 		accounts: Object.freeze( [] ),
 		isVisible: true,
 		onAccountSelected: () => {},
 		service: Object.freeze( {} ),
+		translate: () => {},
```

Comment:
If we later add a component test, let's use [identity](https://lodash.com/docs/4.16.6#identity) instead.